docs/api/platform/flow_control.md

```diff
@@ -2,51 +2,68 @@
 
-Here is an example that uses a ticker object:
+If you don't need the precision of a high-frequency ticker or timeout, we recommend that you use LowPowerTicker or LowPowerTimeout instead. These allow the system to be put in deep sleep mode.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Ticker)](https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Ticker/blob/v6.0/main.cpp)
+- _High-resolution microsecond ticker/timeout_ ([Ticker](ticker.html), [Timeout](timeout.html))
+- _Low Power ticker/timeout_ ([LowPowerTicker](lowpowerticker.html), [LowPowerTimeout](lowpowertimeout.html))
 
-<span class="tips">**Tip:** You may want to read the [power optimization](../apis/platform-concepts.html) tutorial to understand how to achieve power savings. </span>
+Usage of the low power classes will inform the operating system of your desire to allow Deep Sleep mode on your system, although actually entering deep sleep depends on the specific environment and characteristics of your system. For more information about Sleep and Deep Sleep, please refer to our [documentation page](power-management-sleep.html) as well as watch our [Mbed Office Hours video](https://www.youtube.com/watch?v=OFfOlBaegdg&t=12s) where the concepts of Sleep and Deep Sleep are described in-depth.
 
 ### Thread
 
-If your application is running in RTOS mode then threads are another efficient way to blink an LED. During the waiting period, it is possible to take advantage of Mbed OS optimizations to automatically conserve power and deal with other tasks.
+If your application is running in RTOS mode then [Thread](../apis/thread.html)s are another efficient way to blink an LED. During the waiting period, it is possible to take advantage of Mbed OS optimizations to automatically conserve power and deal with other tasks.
 
 [![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Thread)](https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Thread/blob/v6.0/main.cpp)
 
+Threads are the most efficient ways to run tasks in Mbed OS since the operating system internally contains optimizations for efficient scheduling of tasks and maximizing the system's sleep time.
+
+### EventQueue
+
+The [EventQueue](../apis/eventqueue.html) class has a simple-to-use `call_every()` function to schedule repeated actions:
+
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_UsingAPIs/Flow-Control-EventQueue)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_UsingAPIs/Flow-Control-EventQueue/main.cpp)
+
+<span class="tips">**Tip:** For one-off delays, use `call_in()`.
+
+Just as with Ticker/Timeout, if no threads are running during a wait, the system enters sleep mode.</span>
```

Comment:
```suggestion
Just as with Ticker and Timeout, if no threads are running during a wait, the system enters sleep mode.
```

Reply:
Sleep or deep sleep?

Reply:
Mbed OS decides it. I'll change it to "sleep or deep sleep" and add a [link](https://os.mbed.com/docs/mbed-os/v6.0-preview/apis/power-management-sleep.html)